hack/update-codegen-clients.sh

```diff
@@ -27,58 +27,60 @@ BOILERPLATE_HEADER="$( pwd )/hack/boilerplate/boilerplate.go.txt"
 popd
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; go list -f '{{.Dir}}' -m k8s.io/code-generator)}
 
+# "core:v1alpha1 workload:v1alpha1 apiresource:v1alpha1 tenancy:v1alpha1 tenancy:v1beta1 apis:v1alpha1 scheduling:v1alpha1 topology:v1alpha1" \
 bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client" \
-  github.com/kcp-dev/kcp/pkg/client github.com/kcp-dev/kcp/pkg/apis \
-  "core:v1alpha1 workload:v1alpha1 apiresource:v1alpha1 tenancy:v1alpha1 tenancy:v1beta1 apis:v1alpha1 scheduling:v1alpha1 topology:v1alpha1" \
+  github.com/kcp-dev/edge-mc/pkg/client github.com/kcp-dev/edge-mc/pkg/apis \
+  "apis:v1alpha1" \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
   --output-base "${SCRIPT_ROOT}" \
-  --trim-path-prefix github.com/kcp-dev/kcp
+  --trim-path-prefix github.com/kcp-dev/edge-mc
 
 pushd ./pkg/apis
 ${CODE_GENERATOR} \
-  "client:outputPackagePath=github.com/kcp-dev/kcp/pkg/client,apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,singleClusterClientPackagePath=github.com/kcp-dev/kcp/pkg/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
-  "lister:apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
-  "informer:outputPackagePath=github.com/kcp-dev/kcp/pkg/client,singleClusterClientPackagePath=github.com/kcp-dev/kcp/pkg/client/clientset/versioned,apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
+  "client:outputPackagePath=github.com/kcp-dev/edge-mc/pkg/client,singleClusterClientPackagePath=github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned,apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
```

Comment:
Is the swapping of the order of apiPackagePath and singleClusterClientPackagePath here deliberate?

Reply:
ahh, I thought your question was related to the 2 lines having different order.  Your asking, I think, why the order was changed from api, singlecluster to singlecluster, api.  I can reverse the order back.  In the last commit I focused on making line 40 and 42 the same, and brought the singlecluster to first position in both lines.  I will reverse the order in both lines now in a new commit.

Reply:
reverted in latest commit